modules/publicize/publicize-jetpack.php

```diff
@@ -452,7 +452,10 @@ function test_connection( $service_name, $connection ) {
 	 * Save a flag locally to indicate that this post has already been Publicized via the selected
 	 * connections.
 	 */
-	function save_publicized( $post_ID, $post, $update ) {
+	function save_publicized( $post_ID, $post = null, $update = null ) {
+		if ( is_null( $post ) ) {
```

Comment:
Becasue we never use the $post_ID anywhere in that function so it doesn't matter if the value is set.